sdk/digitaltwins/Azure.DigitalTwins.Core/tests/QueryTests.cs

```diff
@@ -124,7 +121,7 @@ await foreach (Page<string> queriedTwinPage in asyncPageableResponse.AsPages())
                 // Test that page size hint works, and that all returned pages either have the page size hint amount of
                 // elements, or have no continuation token (signaling that it is the last page)
                 int pageCount = 0;
-                await foreach (Page<string> page in client.QueryAsync(queryString, options).AsPages())
+                await foreach (Page<string> page in client.QueryAsync(queryString).AsPages(null, pageSize))
```

Comment:
I would suggest using named parameters here so that we can clearly see that "continuationToken" is being set to null.